docs/running_psalm/language_server.md

```diff
@@ -37,10 +37,11 @@ When you install the plugin, you should see a "Language Server Protocol" section
 
 In the "Server definitions" tab you should add a definition for Psalm:
 
+ - Select `Executable`
  - Extension: `php`
- - Path: `<path-to-php-binary>` e.g. `/usr/local/bin/php`
+ - Path: `<path-to-php-binary>` e.g. `/usr/local/bin/php` or `C:\php\php.exe`
     - this should be an absolute path, not just `php`
- - Args: `vendor/bin/psalm-language-server`
+ - Args: `vendor/bin/psalm-language-server` (on Windows use `vendor/vimeo/psalm/psalm-language-server`)
```

Comment:
Does `composer` install `.bat` file into `vendor/bin` instead of a php file on Windows?

Reply:
Exactly. `vendor/bin/psalm-language-server` is a shell script and there is a `.bat` version at `vendor/bin/psalm-language-server.bat`.
Then the LSP plugin will try to execute them using the given PHP interpreter, that just outputs the contents since there is no opening PHP tag `<?php`.
![image](https://user-images.githubusercontent.com/183722/70146136-5f606800-1680-11ea-8add-998d9a31f03e.png)